### PR TITLE
Regiao task

### PR DESCRIPTION
**Pull request by lucasvictor3**
Added Entity, Repository, Service, Controller and tests.

### CODE REVIEW

**Comment by paulofelipefeitosa** [comment]
#11

**Comment by paulofelipefeitosa** [comment]
@lucasvictor3 to improve the code coverage you can, instead of implement tests for RegiaoService, implement tests for the RegiaoController methods.

File: src/test/java/br/com/openenade/api/regiao/RegiaoServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class RegiaoServiceTests {
+
+    @Autowired
+    private RegiaoService service;
```
Comment (paulofelipefeitosa) [inline src/test/java/br/com/openenade/api/regiao/RegiaoServiceTests.java]: You can use RegiaoController instead of RegiaoService :).

Comment (emanueljoivo) [reply src/test/java/br/com/openenade/api/regiao/RegiaoServiceTests.java]: What justifies the use of the Controller instead of a Service?

Comment (paulofelipefeitosa) [reply src/test/java/br/com/openenade/api/regiao/RegiaoServiceTests.java]: @emanueljoivo using the Controllers methods will cover the workflow of the application operations.

**Pull request merged**